libcodechecker/server/client_db_access_handler.py

```diff
@@ -527,6 +553,36 @@ def getRunData(self, run_name_filter):
             session.close()
 
     @timeit
+    def getRunHistory(self, run_id):
+        try:
```

Comment:
Only users who have access to the product should be able to query run history.

```py
self.__require_access()
```
, before the `try` statement.